docs/how-tos/provision.md

```diff
@@ -0,0 +1,310 @@
+
+## Configure `agent-provisioning`
+
+{{< table >}}
+
+{{% tablestep link="/fleet/provision/#configuration" %}}
+**Configure provisioning**
+
+You may optionally provide a provisioning configuration file to customize the experience.
+Place the file at <file>/etc/viam-provisioning.json</file> with the following format:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "manufacturer": "<NAME>", # your company name
+  "model": "<NAME>", # the machine's model
+  "fragment_id": "<ID>", # the fragment id, required for mobile app
+  "hotspot_prefix": "<PREFIX>", # machine creates a hotspot during setup
+  "disable_dns_redirect": true, # disable if using a mobile app
+  "hotspot_password": "<PASSWORD>", # password for the hotspot
+}
+```
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Install `viam-agent`
+
+`viam-agent` is a self-updating service manager that maintains the lifecycle for several Viam services and keeps them updated.
+If you intend to use `viam-agent` to keep your device's Viam software up-to-date or to use its provisioning plugin to let end users set up their own machines, you need to use `viam-agent` and install it on the machine before sending it to the user.
+
+You can [install `viam-agent` on live systems](/installation/) or preinstall it:
+
+{{< tabs >}}
+{{% tab name="Preinstall on an image" %}}
+
+Viam provides a preinstall script that works with images to add `viam-agent`.
+
+{{< alert title="Support notice" color="note" >}}
+Please note this script works only under POSIX (MacOS and Linux) at the moment.
+{{< /alert >}}
+
+{{< table >}}
+{{< tablestep >}}
+**1. Download the preinstall script**
+
+Run the following commands to download the preinstall script and make the script executable:
+
+```sh {class="command-line" data-prompt="$"}
+wget https://storage.googleapis.com/packages.viam.com/apps/viam-agent/preinstall.sh
+chmod 755 preinstall.sh
+```
+
+{{< /tablestep >}}
+{{< tablestep >}}
+**2. Run the script**
+
+Run the preinstall script without options and it will attempt to auto-detect a mounted root filesystem (or for Raspberry Pi, bootfs) and also automatically determine the architecture.
+
+```sh {class="command-line" data-prompt="$"}
+sudo ./preinstall.sh
+```
+
+If this succeeds, continue to step 5.
+Alternatively, go to step 3.
+
+{{< /tablestep >}}
+{{< tablestep >}}
+**3. Run the script for the target system's architecture**
+
+The following command creates a tarball you can extract on your own:
+
+{{< tabs >}}
+{{% tab name="arm64" %}}
+
+```sh {class="command-line" data-prompt="$"}
+sudo ./preinstall.sh --aarch64
+```
+
+{{% /tab %}}
+{{% tab name="x86_64" %}}
+
+```sh {class="command-line" data-prompt="$"}
+sudo ./preinstall.sh --x86_64
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+If this command fails because the script cannot detect your mountpoint, you can specify it directly:
+
+```sh {class="command-line" data-prompt="$"}
+sudo ./preinstall.sh /path/to/rootfs
```

Comment:
This is the backup option for the "SD Card" tab... if it can't find the SD card mountpoint. It likely belongs there if this tab becomes "Create tarball"